spec/requests/preferences/preferences_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+RSpec.describe 'preferences' do
+  describe 'GET #show' do
+    let!(:preference) { create(:preference) }
+
+    context 'when logged in' do
+      let!(:user) { create(:user) }
+
+      before { sign_in user }
+
+      it 'returns a success response' do
```

Comment:
What about checking here that actually you can see all preferences in the index?
This helper [within](https://www.rubydoc.info/gems/capybara/Capybara%2FSession:within) can be useful so you can interate the table.

Reply:
Oka added on feature test